__tests__/app.test.js

```diff
@@ -178,6 +178,7 @@ describe("App", () => {
           expect(review).toHaveProperty("created_at", expect.any(String));
           expect(review).toHaveProperty("votes", expect.any(Number));
           expect(review).toHaveProperty("designer", expect.any(String));
+          expect(review).toHaveProperty("comment_count", expect.any(Number));
```

Comment:
I'd like you to leave this test as it was before, and add an additional test that checks for comment count. Imagine you were working in a dev position, and you completed the first task - we need our code to show that, at the time of writing, that task was exactly what we wanted. Then, maybe your boss comes and asks you to add the comment count functionality - that should be test driven an recognisable as an additional feature of your code.